sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Discount.java

```diff
@@ -34,6 +34,8 @@
     @Column(name = "name")
     private String name;
 
+    private String fileName;
```

Comment:
I think that `imageUrl` or `imagePath` will be better

Reply:
Yes, I think so